src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,105 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int SHIFT_BITS = 16;
+    private Node<K, V>[] table;
+    private int size;
+    private int capacity;
+    private final float loadFactor;
+    private int threshold;
+
+    public MyHashMap() {
+        this.capacity = 16;
+        this.table = (Node<K, V>[]) new Node[capacity];
+        this.loadFactor = 0.75f;
+        this.threshold = (int) (capacity * loadFactor);
+    }
 
     @Override
     public void put(K key, V value) {
-
+        int hashCode = hash(key);
+        int index = Math.abs(hashCode) % table.length;
+        Node<K, V> newNode = new Node<>(hashCode, key, value, null);
```

Comment:
The use of the diamond operator here should be consistent. You should either specify the types or leave both sides with the diamond operator empty.